README.md

```diff
@@ -38,20 +38,26 @@
 <blockquote>Selected Accent: Green</blockquote>
 </details>
 
+
 ## Usage
 
-### Installation
+> [!WARNING]
+> Logseq now officially supports accents for its default theme. Catppuccin Theme **DOES NOT** support the new accent colors.
```

Comment:
Can we raise an issue to support accents or is it just not possible anymore?

Reply:
Not worth it. [Radix colors](https://www.radix-ui.com/colors) is being used under the hood. It would require us to port the Radix colors palette separately and keep supporting the current styles. The app toggles almost all the classnames when accents are used. Almost all other theme creators have taken up this stance and the devs are busy working on a complete rewrite of the app which is going to break the theme anyways when the big update is released.

An ideal solution would be to create a ctp port for radix colors and use it somehow in Logseq or any other app that uses Radix colors.

Reply:
Thanks for the detailed response, that makes sense.
I agree that it would be good to make a radix port and try to ingest that in the future, if possible.